### ISSUE TITLE
Consider renaming because this project isn't ChatGPT

### ISSUE THREAD

**Issue by cartermp** [microsoft/azurechatgpt#104]
ChatGPT is a specific application made up several ML models and lots of features that are different from this application.

This application is a langchain "chat over data" app, that while perhaps quite useful, is not ChatGPT.

The name of this project is not only confusing, but you're actively misleading people in the README by saying things like this:

> Azure ChatGPT is our enterprise option. This is the exact same service but offered as your private ChatGPT.

This is not true. This project is not the exact same service as ChatGPT. It is an alternative kind of application that uses the `/chat` API endpoints.

**Comment by kb-open** [comment]
And if it is really chatgpt, which version of chatgpt is it? README doesn't give any clue!

**Comment by kedoska** [comment]
> And if it is really chatgpt, which version of chatgpt is it? README doesn't give any clue!

This is correct. I had to check [the code](https://github.com/microsoft/azurechatgpt/blob/main/src/features/chat/chat-services/models.ts#L21) to find out that ChatGPT is configurable.

**Comment by tananaev** [comment]
I'm pretty sure that ChatGPT is a trademark that belongs to OpenAI, so you cannot just use that name for a random app, even if it's built on top of OpenAI APIs.

**Comment by marcsalesgrid** [comment]
> > And if it is really chatgpt, which version of chatgpt is it? README doesn't give any clue!
> 
> This is correct. I had to check [the code](https://github.com/microsoft/azurechatgpt/blob/main/src/features/chat/chat-services/models.ts#L21) to find out that ChatGPT is configurable.

The code insinuates that it supports gpt-3.5 and gpt-4. That would be pretty incredible if you could run GPT-4 on your own cloud instance. I thought running GPT-4 requires massive resources?

**Comment by cartermp** [comment]
> The code insinuates that it supports gpt-3.5 and gpt-4. That would be pretty incredible if you could run GPT-4 on your own cloud instance. I thought running GPT-4 requires massive resources?

No, this project does nothing of the sort. It simply calls the existing gpt-3.5 and gpt-4 models via the chat API endpoint, through LangChain. There's nothing self-hostable here, as they'd have to distribute the models and inference servers as well.

Just to reiterate for anyone in this thread -- **this project is not ChatGPT**. It is an open source chat app that integrates with some Azure services so you can do a "chat over your data in Azure" thing. My guess is it's being treated as a reference implementation internally, and not funded as an actual product.
